### PR TITLE
Impoved Exception message string.

### PR DESCRIPTION
**Pull request by cxj**
This new message helps pinpoint the problem and localize it as to its cause.

### CODE REVIEW

**Comment by pmjones** [comment]
Looks like the tests (via TravisCI) fail ... ?

**Comment by cxj** [comment]
Ah, my bad.  Will fix tests today.  Sorry.

**Comment by pmjones** [comment]
No problem; thanks for putting in the PR in the first place. :-)

**Comment by cxj** [comment]
It looks like the remaining problem is that Travis no longer supports PHP 5.3.  Am I reading that correctly?

**Comment by koriym** [comment]
It seems PHP 5.3 support was dropped in Trusty. https://github.com/travis-ci/travis-ci/issues/7163

```
dist: precise
```
Should be work.

**Comment by cxj** [comment]
Thank you, @koriym!

**Comment by cxj** [comment]
Paul, the question to you is do you want to continue supporting PHP 5.3 with this package?  I _think_ it's still possible with some clever Travis configuration to do so, but it's much simpler to drop support for 5.3.  Here's the Travis blog post about the change from Precise to Trusty, and how it affects things:  https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming

**Comment by harikt** [comment]
We solved this on Aura.Html : https://github.com/auraphp/Aura.Html/blob/eda14b491f45fe2e1d1fec3364b1c783d7b7da31/.travis.yml . So copying that travis should work.

**Comment by cxj** [comment]
That's great, Hari!  Thanks much.  I'll add that change to my PR.

**Comment by cxj** [comment]
Now that's confusing.  Why does HHVM fail on one test, while the same test passes for all other versions of PHP?

**Comment by pmjones** [comment]
Freaking HHVM. Well, I can let that one go for now. Merging, and thanks!

**Comment by cxj** [comment]
There's a lot of nuance to the HHVM support, I'm learning.  Travis CI is said to be a rev back on Linux HHVM version, so that makes it even more complicated.

Thanks for the merge.  Would love a new Git tag in near future.  ;-)

File: new.travis.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+language: php
```
Comment (harikt) [inline new.travis.yml:1]: Why do we need this file?

**Review by harikt — COMMENTED** [review]
We can probably make the travis know which platform it is running and skip the code coverage. This will fix the HHVM error `Error:         No code coverage driver is available` .

File: new.travis.yml (hunk 1 above)
Comment (cxj) [reply new.travis.yml:1]: We don't.  It must have gotten swept up in my Git commit locally by accident.

**Pull request merged**